packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1106,4 +1106,67 @@
     </labels>
+    <labels>
+        <fullName>quantic_ComponentError</fullName>
+        <value>{{0}} component error</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>c-quantic-pager component error</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_LookAtDeveloperConsole</fullName>
+      <value>Look at the developer console for more information.</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Look at the developer console</shortDescription>
+    </labels>
+        <labels>
+      <fullName>quantic_InvalidMaxNumberOfDocumentSuggestions</fullName>
+      <value>"{{0}}" is an invalid maximum number of document suggestions. An integer greater than 0 was expected.</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>invalid maximum number of document suggestions.</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_InvalidNumberOfAutoOpenedDocuments</fullName>
+      <value>"{{0}}" is an invalid number of automatically opened document suggestions. A positive integer was expected.</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>invalid number of automatically opened document suggestions.</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_SalesforceFieldNotFound</fullName>
+      <value>The Salesforce field API name "{{0}}" is not found.</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Salesforce field not found.</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_InvalidMaxNumberOfSuggestions</fullName>
+      <value>"{{0}}" is an invalid maximum number of suggestions. A positive integer was expected.</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>invalid maximum number of suggestions.</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_PropertyIsRequired</fullName>
+      <value>The "{{0}}" property is required, please set its value.</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>The property is required.</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_ComponentInitializationError</fullName>
+      <value>Component initialization error</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Component initialization error</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_UnableToInitializeComponent</fullName>
+      <value>Fatal error: unable to initialize the {{0}} component.</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Fatal error: unable to initialize the component.</shortDescription>
```

Comment:
@mmitiche 
Capitalize the first letter after the ":"

"Fatal error: Unable to initialize the component."